docs/includes/apiargs-MongoDBClient-method-startSession-option.yaml

```diff
@@ -0,0 +1,43 @@
+arg_name: option
+name: causalConsistency
+type: boolean
+description: |
+  Enables or disables :ref:`causal consistency <causal-consistency>` for the
+  session. :method:`Mongo.startSession()` enables ``causalConsistency`` by
+  default. After starting a session, you cannot modify its
+  ``causalConsistency`` setting.
```

Comment:
The "Mongo.startSession() enables" sentence looks like it's referring to the shell API, so we can remove that. I also wouldn't bother with the disclaimer that the option cannot be changed later, as all options in the library are permanent.

I think we can leave the first sentence (with its link to the manual) and then copy extra text from the [`MongoDB\Driver\Manager::startSession()`](http://php.net/manual/en/mongodb-driver-manager.startsession.php) docs:

> If true, each operation in the session will be causally ordered after the previous read or write operation. Set to false to disable causal consistency. Defaults to true.